vtex-apps/checkout-payment
# CHK-439: Add installment options modal

#### What problem is this solving?

Moves the installment options from a list inlined in the payment step to a modal. This modal can be accessed right after you fill the credit card data and can be accessed in the following section as well. This PR also adds a "available installment options" modal, so the user can see all the available credit card networks and the installments they have.

I also fixed a minor bug in the credit card iframe, where it's height wasn't being calculated using the correct method, resulting in a "over-heighted" (if that word exists...) iframe and causing some layout issues on mobile.

#### How should this be manually tested?

[Workspace](https://lucas--checkoutio.myvtex.com/cart/add?sku=289).

You can see the aforementioned changes in the payment step of the above workspace.

#### Checklist/Reminders


#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements
